scripts/install

```diff
@@ -14,7 +14,9 @@ popd
 pushd vendor/github.com/golang/protobuf
 go install ./protoc-gen-go
 popd
-
 go run resources/installBlockchainDependencies/main.go
 go generate ./...
 popd
+cd escrow
+protoc -I . ./*.proto --go_out=plugins=grpc:.
```

Comment:
Is it really needed? Line 18 above should generate all necessary protobuf stubs.